src/components/00-materials/README.md

```diff
@@ -74,20 +74,24 @@ this.render() {
 | XingSvg          |
 | YoutubeSvg       |
 
-| images         |
-| -------------- |
-| AxaLogoSvg     |
-| AxaLogoOpenSvg |
+| images                          |
+| ------------------------------- |
+| AxaLogoSvg                      |
+| AxaLogoOpenSvg                  |
+| ... and many more in subfolders |
 
 ## Contribution
 
 ### Process:
 
 1. Involve a designer by sending him the new SVG in question, unless the designer already has the file. He will review it and possibly change some things, for example by adjusting inner padding etc. in order to guarantee overall quality.
 2. Remove unnecessary code inside the SVG:
-    - a. `data-name="Layer 2"` # layer info for vector drawing software
-    - b. `<path fill="#fff" d="M0 0h96v96H0z"/>` # outer bounding box filled with white background
-3. 
-    - a. use the internal [svgo](https://github.com/svg/svgo)-invoking scripts `npm run build-icons` or `npm run build-images` to clean up and optimize the SVGs, then copy the optimized files back from the `.tmp` folder to `icons-raw` or `images-raw`
-    - b. alternatively use the [online SVG optimizer](https://jakearchibald.github.io/svgomg/] for the same purpose
+   - a. `data-name="Layer 2"` # layer info for vector drawing software
+   - b. `<path fill="#fff" d="M0 0h96v96H0z"/>` # outer bounding box filled with white background
+3. - a. use the internal [svgo](https://github.com/svg/svgo)-invoking scripts `npm run build-icons` or `npm run build-images` to clean up and optimize the SVGs, then copy the optimized files back from the `.tmp` folder to `icons-raw` or `images-raw`
+   - b. alternatively use the [online SVG optimizer](https://jakearchibald.github.io/svgomg/] for the same purpose
 4. Manually review the SVGs produced in the last step: add or edit &lt;path&gt; attributes, setting `fill="currentColor"` and/or `strokes="currentColor"` as appropriate. Remove unnecessary attributes.
+
+## ChangeLog
+- 2.0.0
+    - removed Axa Logos from /icons
```

Comment:
And you moved it to images, right? Or was it duplicated all along?

Reply:
It was a duplicate. Just some technical differences. We have to release materials as a breaking change because of that.

Reply:
Jep, pushed that right now